pl_examples/bug_report_model.py

```diff
@@ -53,13 +53,13 @@ def run():
     model = BoringModel()
     trainer = Trainer(
         default_root_dir=os.getcwd(),
-        limit_train_batches=1,
-        limit_val_batches=1,
+        limit_train_batches=30,
+        limit_val_batches=30,
         num_sanity_val_steps=0,
-        max_epochs=1,
+        max_epochs=3,
```

Comment:
can we keep these to 1? It's useful for debugging to just have one single step.

Reply:
Is it tho? I always increase it when I use it for debugging ahha (also, the steps are tiny)

Reply:
Updated!